### PR TITLE
Fixed color of some icons on components and exported colors enum

### PR DESCRIPTION
**Pull request by tiagoalmeidadarosa**
Before submitting this PR, please make sure:

- [x] You review intended branch to merge into
- [x] You set a meaninful Title 
- [ ] You associated WI
- [x] You added reviewers
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added/run unit tests

### CODE REVIEW

File: src/components/Icons/UploadFile/index.tsx [hunk 1]
```diff
@@ -1,37 +1,72 @@
 import * as React from 'react';
 
-const UploadFile: React.FC<React.SVGProps<SVGSVGElement>> = ({ ...props }) => {
-  // eslint-disable-next-line no-console
-  console.warn('Icon.UploadFile is deprecated, please use Icon.Upload instead.');
```
Comment (EduardoUmpierre) [inline src/components/Icons/UploadFile/index.tsx:5]: Isn't this icon deprecated anymore?

File: src/index.tsx [hunk 2]
```diff
@@ -100,4 +101,5 @@ export {
   Mixins,
   Range,
   SmallInput,
+  Themes,
```
Comment (EduardoUmpierre) [inline src/index.tsx:104]: Non-blocking: should we export the Styled-components' ThemeProvider to allow the theme customization without requiring the application to have Styled-components as a dependency?

File: src/components/Dropdown/styles.ts [hunk 3]
```diff
@@ -90,3 +91,8 @@ export const List = styled.ul<ListProps>`
     }
   }
 `;
+
+export const Button = styled(CustomButton)`
```
Comment (EduardoUmpierre) [inline src/components/Dropdown/styles.ts:95]: The old `Button` from `Dropdown/index.stories.tsx` was based on the `CustomButton` component and not only on its style.

Is this what you want?

File: src/components/Icons/UploadFile/index.tsx (hunk 1 above)
Comment (EduardoUmpierre) [reply src/components/Icons/UploadFile/index.tsx:5]: This icon isn't the same as the Icon.Upload as it has a cloud shape. Resolving this comment.

Comment (tiagoalmeidadarosa) [reply src/components/Icons/UploadFile/index.tsx:5]: This icon has never been exported... it is only used within the upload component

File: src/components/Dropdown/styles.ts (hunk 3 above)
Comment (tiagoalmeidadarosa) [reply src/components/Dropdown/styles.ts:95]: CustomButton with variant 'link' does not accept children that are just text, in this case an icon is needed after the text of the button. It was the way I found to solve for now

**Review by EduardoUmpierre — APPROVED** [review]

**Pull request merged**